### PR TITLE
Fix seat colour of TD4 River Rapids, fixes #1695

### LINKED ISSUE #1695: TD4 Import [Ropey Rapids.TD4] River Rapids colour bug.
When I import the Ropey Rapids.TD4 in OpenRCt2 than it has a colour bug on the boats.

In RCT1 they were Black/Red while there are now in OpenRCt2 Green/Red

TD4 file:
https://drive.google.com/file/d/0BzdfIT8snZiGdGJzYklFTUtGTW8/view?usp=sharing

### PR DESCRIPTION
**Pull request by Gymnasiast**

### CODE REVIEW

File: src/ride/track.c [hunk 1]
```diff
@@ -646,9 +646,14 @@ rct_track_td6* load_track_design(const char *path)
 		}
 
 		memcpy(&track_design->vehicle_object, vehicle_object, sizeof(rct_object_entry));
-		for (i = 0; i < 32; i++)
+		for (i = 0; i < 32; i++) {
 			track_design->vehicle_additional_colour[i] = track_design->vehicle_colours[i].trim_colour;
 
+			// RCT1 river rapids always had black seats.
+			if (track_design->type == RCT1_RIDE_TYPE_RIVER_RAPIDS)
```
Comment (RobertWHurst) [inline src/ride/track.c]: Wait, so the trim color must be black now on all river rapids?

Comment (IntelOrca) [inline src/ride/track.c]: No, this is just a colour fix for when you import RCT1 TD4 tracks.

Comment (Gymnasiast) [inline src/ride/track.c]: Oh, come on! You would have known this if you had looked at the context: this is import code for TD4 designs, which are from RCT1. RCT1 had no adjustable trim colour for river rapids, instead the trim was always coloured black.

Comment (RobertWHurst) [inline src/ride/track.c]: Ah, I see. Cool :sunglasses:

**Pull request merged**